mmdet/models/roi_heads/point_rend_roi_head.py

```diff
@@ -131,23 +162,32 @@ def simple_test_mask(self,
         """Obtain mask prediction without augmentation."""
         ori_shapes = tuple(meta['ori_shape'] for meta in img_metas)
         scale_factors = tuple(meta['scale_factor'] for meta in img_metas)
+
+        if isinstance(scale_factors[0], float):
+            logger.warning(
+                'Scale factor in img_metas should be a '
+                'ndarray with shape (4,) '
+                'arrange as (factor_w, factor_h, factor_w, factor_h), '
+                'The scale_factor with float type has been deprecated. ')
+            scale_factors = np.array([scale_factors] * 4, dtype=np.float32)
```

Comment:
Why don't we directly make it a torch tensor here?

Reply:
1. In line 224, there is `torch.from_numpy(scale_factor)` which needs `scale_factors` to be np.ndarray.
2. This is copied from `standard_roi_head`.